scripts/load-latest-premium-docker-image.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+#===================================================================================
+# Usage:
+#   ./load-latest-premium-docker-image.sh
+#
+# Description:
+#   The script has been created to provide a tool for downloading and installing cloudify docker image with one command.
+#   It is mainly used in Makefile inside of cloudify-stage and cloudify-blueprint-composer repositories. It can be also used separately in aforementioned projects.
```

Comment:
Let's maybe fit it in that ASCII box 🙂 

```suggestion
#   The script has been created to provide a tool for downloading and installing 
#   cloudify docker image with one command. It is mainly used in Makefile 
#   inside of cloudify-stage and cloudify-blueprint-composer repositories. 
#   It can be also used separately in aforementioned projects.
```